src/api/flaskr/api/llm/__init__.py

```
@@ -367,12 +367,28 @@ def _load_gemini_models(
     return models
 
 
+def _load_deepseek_models(
+    config: ProviderConfig, params: Dict[str, str], base_url: Optional[str]
+) -> List[Union[str, Tuple[str, str]]]:
+    api_key = params.get("api_key", "")
+    try:
+        return _fetch_provider_models(api_key, base_url)
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)